docs/BacwardCompatability.md

```diff
@@ -0,0 +1,51 @@
+# Compatability Requirements and Implementation Plan
+
+## What is Compatability
+Before talking about how to provide compatability, it is worthwhile to clarify what we mean by compatibility. 
+Fybrik required to provide forwards and backwards compatibility in its behaviour. Compatibility is hard, especially 
```

Comment:
is required

Reply:
What is forwards compatibility?

Reply:
> is required

done

>What is forwards compatibility?

You can see examples of forward and backward compatibility at the reference 5.